dev/com.ibm.ws.jsf.2.3_fat/test-applications/CDICommon/src/com/ibm/ws/jsf23/fat/cdi/common/beans/jsf23/ActionListenerBean.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package com.ibm.ws.jsf23.fat.cdi.common.beans;
+package com.ibm.ws.jsf23.fat.cdi.common.beans.jsf23;
```

Comment:
Since the file didn't change (another have a package rename) it doesn't need a copyright update, correct?

Reply:
Correct, just moving the file.